### ISSUE TITLE
Templating?

### ISSUE THREAD

**Issue by sashabeep** [vanilophp/docs#22]
Can't find any templating guide. I'm not a programmer and it's very difficult to understand how to do anything. I'm widely familiar with blade template engine but that's it.

Some helpers needed for building menus / static pages / product list collections by criteria etc...

**Comment by fulopattila122** [comment]
Hi Sasha,

**tl;dr**

There's no templating guide because Vanilo doesn't ship with any controllers/views, etc.
That is something that the developers have to build themselves using blade templates, including the making of the backend routes, controllers and fetching and injecting the required data for the blade views.

There is a sample application that has most of the stuff you mention here: https://github.com/vanilophp/demo#vanilo-demo-application

---

Please note that Vanilo is an E-commerce **Framework** and not an E-commerce *Application*. The difference between a framework and an application is that frameworks are very flexible and are suitable for building virtually all kinds of e-commerce-related (web) applications.

An **e-commerce framework**:
- consists of reusable components
- is highly customizable
- contains bootstrap code to start the application
- but is not an application itself
- **programming knowledge is required**

Whereas an **e-commerce application** is:
- a ready-made system that works out of the box,
- it is configurable,
- usually less flexible,
- **programming knowledge is not necessary**

As a consequence, using **Vanilo requires programming knowledge**.

If you're looking for a good alternative that doesn't need programming knowledge, check out Bagisto: https://bagisto.com/en/

**Comment by sashabeep** [comment]
@fulopattila122 Thank you for detailed explanation, it was very helpful indeed
